pkgs/misc/drivers/utsushi/default.nix

```
@@ -0,0 +1,144 @@
+{stdenv, fetchurl, autoreconfHook, boost, gnome2,
+pkg-config, libtool, udev, libjpeg, file, texlive,
+libusb, libtiff, imagemagick, sane-backends,
+}:
+
+/*
+Alternatively, this package could use the "community source" at 
+https://gitlab.com/utsushi/utsushi/
+Epson provides proprietary plugins for networking, ocr and some more
+scanner models. Those are not (yet ?) packaged here.
+*/
+
+stdenv.mkDerivation rec {
+  pname = "imagescanv3";
+  version = "3.57.0";
+
+  src = fetchurl {
+    url = "http://support.epson.net/linux/src/scanner/imagescanv3/common/imagescan_${version}.orig.tar.gz";
+    sha256 = "0qy6n6nbisbvy0q3idj7hpmj9i85cd0a18klfd8nsqsa2nkg57ny";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    autoreconfHook
+  ];
+  buildInputs = [
+    libtool
+    boost
+    libusb
+    libtiff
+    libjpeg
+    udev
+    imagemagick
+    sane-backends
+    gnome2.gtkmm
```

Aliases should not be used in nixpkgs

```suggestion
    gtkmm2
```